deploy/handler/operator.yaml

```diff
@@ -216,6 +216,13 @@ spec:
               mountPath: /var/k8s_nmstate
           securityContext:
             privileged: true
+          readinessProbe:
+            exec:
+              command:
+              - nmstatectl
+              - show
+            initialDelaySeconds: 5
+            periodSeconds: 5
```

Comment:
This means we will stop retrying this once it succeeds, since it is readiness, right?

Reply:
Yep.